### ISSUE TITLE
Trouble Installing

### ISSUE THREAD

**Issue by ceporras2** [ugswork/eaudit#2]
Hello, 
I wanted to experiment with the eAudit tool but was having trouble understanding the installation process. Are we to clone the repo in any linux distro and then just run the bcc_install.sh file for the tool to installed?
If anybody could help it would be greatly appreciated. Thank you!

**Comment by profsekar** [comment]
Yes. I thought this is stated in the README file. If there is something
that can be improved in that file, please let me know. We don't have the
manpower to offer support beyond occasional questions over email.
Best,
-- Sekar

**Comment by ceporras2** [comment]
@profsekar I was able to get it cloned and installed into my environment. 
However, now that I have it installed I am wanting to test it and view the logs that are produced by eAudit, but I'm not sure where to look to find the logs or how to be sure it is running and recording. 
Could you please help clarify this or point to some tutorial for the basic functionality?

**Comment by profsekar** [comment]
In addition to the README, there is documentation in the form of usage
message. As a research tool for interested users --- and not a product that
people pay for --- it does require some effort on the part of the users.
